### DISCUSSION TITLE
Save spam log as per the channel name then inculde time

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [ThioJoe/YT-Spammer-Purge#587] [Ideas]
Rn the spam log name is as follows:
Spam_Log_2022-02-07_11-50-13

It would be better if the log name would be {Spam_Log_<channel which was scanned>_2022-02-07_11-50-13 (time)

**Comment by Firecul** [comment]
`11-50-13` **is** the time

**Comment by KendallDoesCoding** [comment]
yup ik

**Comment by Firecul** [comment]
Then I'm very confused as to why you are suggesting the log name to add the time again?

**Comment by KendallDoesCoding** [comment]
Yeah, sorry about that. Just it should say the channel name in the log.

**Comment by Firecul** [comment]
It would need a log for every single video scanned then because if people give a csv list of urls they won't all be from the same channel probably.

**Reply by KendallDoesCoding** [reply]
True.

**Reply by Firecul** [reply]
That's also why my similar suggestion of a folder structure ~\\{Channel}\\{Video}\\{Scan date and time} would maybe help this idea but would take a decent amount of work I think.

**Reply by ThioJoe** [reply]
> It would need a log for every single video scanned then because if people give a csv list of urls they won't all be from the same channel probably. 

Actually the way the program works, it only allows scanning videos from one channel at a time anyway

**Reply by Firecul** [reply]
> Actually the way the program works, it only allows scanning videos from one channel at a time anyway

I've never used that particular feature so I didn't know that, good to know.
